plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/json/HostConfig.java

```diff
@@ -41,6 +41,7 @@
     private String         pidMode;
     private boolean        readonlyRootfs;
     private Ulimit[]       ulimits;
+    private String         volumeOptions;
```

Comment:
I don't see such field in Docker Create container API. Can you point me out where it is?